src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -139,7 +139,7 @@ private void InitManifestModule()
 
         #region DefineDynamicAssembly
 
-        [RequiresDynamicCode("Generating new code at runtime is not supported with native AOT.")]
+        [RequiresDynamicCode("Creating a DynamicMethod requires dynamic code.")]
```

Comment:
These two aren't for creating a DynamicMethod:

```suggestion
        [RequiresDynamicCode("Defining a dynamic assembly requires dynamic code.")]
```

Reply:
Thanks